src/network-config.js

```diff
@@ -160,3 +172,12 @@ export function sanitizeNetworkType(networkType) {
   }
   return networkType
 }
+
+export function getWalletConnectRPC(chainId = -1) {
```

Comment:
Using it instead of `getWalletConnectRpc()` it's useless and break architectural patters

Reply:
But it's different, the reason why fortmatic and portis are using helpers to get their configuration is because their ID/key are supposed to be private, whereas my understanding is that in the case of wallet connect the rpc url should always point to the aragon node of that specific chain.
the only reason why i added a helper for walletconnect as well is to initialize it with the correct chain on the react context without having to destructure the entire config file
https://github.com/OpiumProtocol/client/blob/2a01205ab4864947ad3ace2a6bebd2a89411bd77/src/wallet.js#L92

Reply:
Got you, makes sense